diff --git a/node_modules/@jis3r/icons/dist/icons/binary.svelte b/node_modules/@jis3r/icons/dist/icons/binary.svelte
index 5434ee5..d05d42e 100644
--- a/node_modules/@jis3r/icons/dist/icons/binary.svelte
+++ b/node_modules/@jis3r/icons/dist/icons/binary.svelte
@@ -46,7 +46,7 @@
 		stroke-linecap="round"
 		stroke-linejoin="round"
 		class="binary-icon"
-		class:animate={hovered}
+		class:animate={hovered || isHovered}
 	>
 		<rect x={14} y={14} width={4} height={6} rx={2} class="binary-rect1" />
 		<rect x={6} y={4} width={4} height={6} rx={2} class="binary-rect2" />
